music-suite.cabal

```diff
@@ -66,7 +66,8 @@ library
                         random,
                         neat-interpolation,
                         modular-arithmetic,
-                        random-shuffle
+                        random-shuffle,
+                        optparse-applicative
```

Comment:
This is already listed higher up in the file :).

Reply:
whoops